app/templates/items/detail/tabs/_item_options.hbs

```diff
@@ -0,0 +1,130 @@
+            {{/if}}
+          </div>
+        </div>
+      </div>
+
+      <div class="columns small-6 tab sub-block main">
+        <div class="text-block row divided">
+          <div class="subtab column small-4">
+            <div class="line icon-holder">{{fa-icon 'paper-plane' size="lg"}}</div>
+            <div class="line text-holder"> {{ item.totalDispatchedQty }} </div>
+          </div>
+          <div class="subtab column small-4">
+            <div class="line icon-holder">{{fa-icon 'shopping-basket' size="lg"}}</div>
+            <div class="line text-holder"> {{ item.totalDesignatedQty }} </div>
+          </div>
+          <div class="subtab column small-4">
+            <div class="line icon-holder">{{fa-icon (if item.allowWebPublish 'eye' 'eye-slash') size="lg"}}</div>
+            <div class="line text-holder"> {{ item.availableQty }} </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="order-preview">
+      <div class="row gc-orders-package-block">
+        <div class="columns small-3 large-2 left">
+          <img src={{item.thumbImageUrl}} class="thumb" />
+        </div>
+
+        <div class="columns small-9 large-10 sub-block main" >
+          <div class="text-block">
+            <div class="line">{{ item.code.name }}</div>
+            <div class="line">{{ item.notes }}</div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="form-container">
+      <div class="title">{{fa-icon actionIcon}} {{actionName}}</div>
+      <div class="infos">
+        <div class="info-row">
+          <div class="key">Item</div>
+          <div class="value">{{fa-icon 'tag'}} {{item.inventoryNumber}}</div>
+        </div>
+
+        <div class="info-row">
+          <div class="key">From Location</div>
+          <div class="value clickable" {{action "selectLocationAction"}}>{{fa-icon 'map-marker-alt'}}
+            {{actionFrom.displayName}}</div>
+        </div>
+
+        <div class="info-row">
+          <div class="key">Quantity</div>
+          <div class="value">
+            {{input type="number" value=actionQty class="qty-input"}}
+          </div>
+          <div class="sub-text {{if (is-not (is-in-range actionQty 1 maxQuantity)) 'warn-text'}}">Max: {{maxQuantity}}
+          </div>
+        </div>
+
+        <div class="info-row item-action-comment">
+          <div class="key">Comment</div>
+          <div class="value">
+            {{auto-resize-textarea
+                data-autoresize=true
+                name="actionComment"
+                value=actionComment
+                id="item-action-comment"
+                maxlength="180"}}
+          </div>
+        </div>
+
+      </div>
+    </div>
+
+    <div class="button-container">
+      <div class="button light" {{action "cancelAction"}}>Cancel</div>
+      {{#if (is-not (is-in-range actionQty 1 maxQuantity))}}
+      <div class="button" disabled>{{actionName}} ({{ actionQty }})</div>
```

Comment:
May be this indentation problem is due to config issue in editor, i've seen shreyash and akshay had also faced the same.